apps/src/blockly/addons/cdoGesture.js

```diff
@@ -0,0 +1,25 @@
+export function overrideHandleTouchMove(blocklyWrapper) {
+  const cdoGesture = blocklyWrapper.Gesture.prototype;
+
+  // Override handleTouchMove function
+  cdoGesture.handleTouchMove = function (e) {
```

Comment:
I was thinking about this a little bit and if you do not want to rely too much on duplicating their code, which might pose a problem later, you might just patch out the recursion. It seems their bug is related to not turning off '`isMultiTouch`' before calling into `handleMove` because they intend to 'emulate' a normal move when they do not detect a gesture.

It really only needs to be:

```
const cdoGesture = blocklyWrapper.Gesture.prototype;
const oldHandleTouchMove = cdoGesture.handleTouchMove;

cdoGesture.handleTouchMove = function(e) {
  this.isMultiTouch_ = false;
  return oldHandleTouchMove.bind(this)(e);
};
```

but not entirely sure.

Reply:
For the sake of documentation, leaving a note here that this approach did not quite work. While it prevents the errors, it results in some glitchy behavior where the dragging block bounces back and forth between each touch point.